net-im/bitlbee-facebook/bitlbee-facebook-1.1.2.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils autotools
+
+DESCRIPTION="Facebook protocol plugin for BitlBee"
+HOMEPAGE="https://github.com/bitlbee/bitlbee-facebook"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="debug"
+
+if [ "${PV}" = "9999" ] ; then
```

Comment:
It'd be better to not have a conditional SRC. We should really limit how much code is run in the global space.

Reply:
This fine and absolutely standard in the Gentoo tree, no reason to change.
Still, use bash and not POSIX: `if [[ ${PV} == *9999 ]] ; then`